src/handlers/invite/mod.rs

```diff
@@ -19,14 +22,14 @@ pub struct InviteKeyClaims {
 
 impl JwtClaims for InviteKeyClaims {
     fn is_valid(&self) -> bool {
-        true
         // TODO: Add validation:
         // aud must be equal this dns?
         // exp must be in future
         // iat must be in past
-        // iss must be valid did:key
         // sub must be valid public key
         // pkh must be valid did:pkh
+
+        did::validate_x25519(&self.iss)
```

Comment:
Oh, just noticed. istead of `iss` field it should be `sub` field. `iss` Is Identity Key and that key is of Ed25519 type